plsync/slices.py

```diff
@@ -90,7 +90,9 @@
                                                 Attr('MeasurementLabCentos', capabilities='CAP_NET_BIND_SERVICE,vxc_^28'), ], 
                                            users=user_list,
                                            use_initscript=True,
-                                           ipv6="nuq1t nuq02 lga06" ),
+                                           ipv6=['mlab1.nuq1t', 'mlab2.nuq1t', 'mlab3.nuq1t', 'mlab4.nuq1t',
+                                                 'mlab1.nuq02', 'mlab2.nuq02', 'mlab3.nuq02', 'mlab4.nuq02',
```

Comment:
Does nuq02 have 4 nodes?  mlab4.nuq02.measurement-lab.org doesn't resolve, though the sites spreadsheet lists nuq02 as having 4 nodes.  Also, there is no mlab4.nuq02 in PLC.  For now, I think it's probably best to remove mlab4.nuq02 from this list.